web/themes/custom/liibarrio2020/scss/components/_view.scss

```diff
@@ -487,6 +487,16 @@
 
 //Judgment Listings Edits//
 
+
+//Hides content types without a subject matter type
 .view-judgments-listing .view-display-id-subject_group .views-summary li:first-child {
   display: none;
+
+}
+.view-judgments-listing .view-header .h3 {
+ padding-top: 10px;   
+}
+
+.view-judgments-listing .view-content h3 {
+    padding-top: 10px;
```

Comment:
Are you sure you need both `.h3` and `h3` styled?

Also, SCSS makes this simple and clearer that very similar elements are getting the same styling:

```scss
.view-judgments-listing .view-header {
  h3, .h3 {
    padding-top: 10px
  }
}
```